lib/geometry/polygon.nit

```diff
@@ -151,7 +139,38 @@ class ConvexPolygon
 		return projection
 	end
 
-	# Is this polygon convex ?
+	# Remove  `p` from the vertices, keeping at least 3 vertices
+	fun delete_vertex(p: Point[Float]) do if points.length > 3 then points.remove(p)
```

Comment:
The no-op is not POLA. Maybe a contract (therefore an assert) or simply handle the case of polygons with less that 3 points.